predictoor/README.md

```diff
@@ -0,0 +1,45 @@
+
+Alas, accurate predictions are *hard*. Worse, typical prediction feeds have no accountability on accuracy. If the weatherman says "no rain for today" and then it rains, a farmer could get stuck in the mud, wrecking a portion of his crops. The weatherman doesn't feel the impact of wrong predictions, but the farmers sure care! 
+
+**Accountable Predictions** - Imagine if there was accountability. Accuracy would go up; the farmer would be stuck less. Imagine accountable prediction feeds for not only for rain, but also wind, sea temperature, road congestion, train delays, ETH prices, NVID prices, housing prices, and more. **Imagine tens of thousands of prediction feeds with accountable accuracy.** Imagine them **globally distributed**, and censorship resistant. Imagine **accuracy improving with time**. 
+
+## Introduction
+Predictoor is composed of different actors, often referred to as: predictoors and traders. You will learn in the [Introduction To Predictoor](#introduction) a lot more about these actors, how they are structured in the system, and their behaviors. By the time you've completed reading this, you'll know what Epochs are, and have a better understand of what "t+1" means.
+
+## Architecture
+After obtaining a broad overview of the system and it's actors, you should be ready to dive into [The Architecture of Predictoor](#architecture) and the repository. You'll become more exposed to the key building blocks and pdr-backend, the integration component for Predictoors and Traders.
+
+We anticipate Predictoor to extend beyond DeFi to other verticals like climate and agriculture. Predictoor agents may evolve into AI DAOs with emergent swarm-like behavior. This is the future.
+
+## Earning with Predictoor
+Predictoor serves two groups of actors: Predictoors, and Traders. In this section, we'll explore [How to Earn with Predictoor](#earning-with-predictoor) and describe how both of these groups can benefit from it. After you are done with this, you should know what steps you need to take next in order to start leveraging the system for your own benefit.
+
+We'll also provide some links and exercise to make this easier for you.
+
+## Predictoor Parameters and Roadmap
+[Predictoor Parameters](#predictoor-parameters) will provide such as incentives, feed costs, deployed feeds, and details about it's possible future.
+
+## Connect with Us
+If you want to talk to other Predictoors, Traders, and contributors to this ecosystem, [Join us on Discord](https://discord.gg/TnXjkR5).
+
+## Predictoor Resources
+These documents are a summary of many resources.
+- Blog Post: ["Meet Predictoor: Accountable Accurate Prediction Feeds"](https://blog.oceanprotocol.com/meet-predictoor-accountable-accurate-prediction-feeds-8b104d26a5d9)
```

Comment:
`Blog Post` -> `Main blog post`

Add new line:
`- All blog posts: ["Ocean Predictoor Series"](https://blog.oceanprotocol.com/ocean-predictoor-series-6a9122754a4d)`

Why: now as we add more blog posts, we don't have to update docs all the time. (FYI I just created the "Ocean Predictoor Series" post. It's like the DF equivalent. Super useful for reference.)

Reply:
Thanks for linking to this. Done.